src/index.html

```diff
@@ -10,7 +10,47 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="dig"
+      class="searchBar"
+      action="/"
+      method="get"
+    >
+      <!-- <img
+      src="./images/Search.svg"
+      alt="search"
+      class="ButtonBig"
+    > -->
+      <input
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+      data-qa="keypress"
+      class="searchImputbig"
+    >
```

Comment:
```suggestion
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
        class="searchImputbig"
      >
```
fix code style also in other places